src/App.js

```diff
@@ -35,136 +37,146 @@ import {
   isVersionAllowedUpdate,
   updateServerInfo,
 } from './actions/index';
-import { connect } from 'react-redux';
 import versionApi from './api/version';
 import helpers from './utils/helpers';
-import { axios as hathorLibAxios, config as hathorLibConfig } from '@hathor/wallet-lib';
 import { BASE_URL } from './constants';
 import createRequestInstance from './api/customAxiosInstance';
 
 hathorLibConfig.setServerUrl(BASE_URL);
 
-const mapDispatchToProps = dispatch => {
-  return {
-    dashboardUpdate: data => dispatch(dashboardUpdate(data)),
-    isVersionAllowedUpdate: data => dispatch(isVersionAllowedUpdate(data)),
-    apiLoadErrorUpdate: data => dispatch(apiLoadErrorUpdate(data)),
-    updateServerInfo: data => dispatch(updateServerInfo(data)),
-  };
-};
+const NavigationRoute = ({ internalScreen: InternalScreen }) => (
+  <div>
+    <Navigation />
+    <InternalScreen />
+  </div>
+);
```

Comment:
quest(if-minor): Why did we remove `Route` from the `NavigationRoute`?

Since all references of this are inside a `Route` we could just include it here, this makes the router more readable.
What do you think?

Reply:
The reason for this is that the structure of Routes in React Router v6 changes, and the guide to migrate from v5.1 recommends this as an intermediate step towards making the upgrade easier.

In short, the idea is that a route should only contain props and data about routing, and not anything related to the business rules of the application. You can read more about it on the [official migration docs](https://reactrouter.com/en/main/upgrading/v5#refactor-custom-routes).